src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectMessage.cs

```diff
@@ -535,6 +543,11 @@ public string Sid
         }
 
         /// <summary>
+        /// Gets 'skuTelemetryValue'.
```

Comment:
Comment should read as an English statement. Something like: Gets the string value that is used when sending telemetry data as part of an OpenIdConnectRequest message.

Reply:
Done!